auth.go

```diff
@@ -411,10 +413,16 @@ func (x *Central) Login(identity, password string) (sessionkey string, token *To
 		err = authErr
 		x.Stats.IncrementInvalidPasswords(x.Log)
 		x.Log.Infof("Login Authentication failed (%v) (%v)", identity, err)
+
+		// Send failed login to audit service
+		auditUserAction(identity, password, "Login Authentication failed")
 		return sessionkey, token, err
 	}
 	x.Log.Infof("Login authentication success (%v)", userId)
 
+	// Send success login to audit service
+	auditUserAction(identity, password, "Login Authentication success")
```

Comment:
We need to handle error in both cases. I am not sure if we should log it as an actual ERROR or WARNING if we cannot hit the audit service or their was any problem. Discuss how critical this is with @bmharper. I assume a Warning is enough, and if the user successfully logged in, things should flow as normal. If you only need to log a warning, you can just do it in the function itself.